functorch/_src/eager_transforms.py

```diff
@@ -609,6 +605,11 @@ def assert_flat_tuple_of_tensors(elts, api, argname):
             f'{api}: Expected {argname} to be a non-empty tuple of Tensors.')
 
 
+def assert_non_empty_output(output, api):
```

Comment:
nit: we should write in a comment that output must be a List (or use type annotations, but we haven't really been doing that).

Reply:
I can start with type annotations :)